chat-client/package.json

```diff
@@ -11,9 +11,8 @@
     "license": "Apache-2.0",
     "scripts": {
         "compile": "tsc --build && npm run package",
-        "test:prettier": "prettier . --check",
```

Comment:
doesn't this mean local builds and tests will ignore formatting errors? Or so we still have a mechanism against this

Reply:
We now rely on a different mechanism for these - precommit hooks. So locally you won't be able to commit unless everything is formatted and pretty.